core/request.py

```diff
@@ -288,7 +289,6 @@ def processRequest(self) -> None:
             # Set machine-readable x-viur-error response header in case there is an exception description.
             if e.descr:
                 self.response.headers["x-viur-error"] = e.descr.replace("\n", "")
```

Comment:
Re-adding removed newline
```suggestion
                self.response.headers["x-viur-error"] = e.descr.replace("\n", "")

```